docs/General/Manuals/client-manual.rst

```diff
@@ -404,6 +404,91 @@ consists of the following items:
 Game Menu
 ^^^^^^^^^
 
+The :guilabel:`Game` menu is used to conduct global level options on the client or the game being played.
+
+The :guilabel:`Game` menu has the following options:
+
+Save Game
+    Saves the game as it is :strong:`right now` to the ``saves`` directory in the user's profile directory.
+    On Linux and Mac OS, the profile directory is inside the User's home directory such as
+    ``~/.local/share/freeciv21``. On Windows the profile directory is inside the User ``AppData`` directory
+    such as ``C:\\Users\\<userName>\\AppData\\Roaming\\freeciv21``.
+
+Save Game As ...
+    Saves the game as it is :strong:`right now` to a location of your choosing.
+
+Save Map to Image
+    Save a ``.png`` image file of the map to the user's profile directory.
+
+Set Local Options
+    Opens the :guilabel:`Local Options` dialog box as described in the `Options`_ section above.
+
+Server Options
+    Opens the :guilabel:`Server Options` dialog as described in the `More Game Options`_ section
+    above.
+
+Messages
+    Opens the :guilabel:`Message Options` dialog as shown in Figure 10 below. The screenshot only shows a few
+    rows of available options, many more will be found in the client. Any item with a check mark in the
+    :guilabel:`Out` column will be shown in the :guilabel:`Server Log/Chat` widget. Any item with a check mark
+    in the :guilabel:`Mes` column will be shown in the :guilabel:`Messages` widget (see `Messages`_). Lastly,
+    any item with a check mark in the :guilabel:`Pop` column will be shown in a Pop-Up Message Box window.
```

Comment:
Maybe a warning here about setting this to `pop`, since it can crash the client if too many things popup.

Reply:
Well... a crash is a bug